Bottom: 9a878aabd39520f704333e664e41d87a73a56498
Top:    f662f07843423b221fb61150ec1deffe3981a1c8
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2012-11-20 02:03:38 +0000

autoroute.c: Destroy debug graphics context when we're finished with it


---

diff --git a/src/autoroute.c b/src/autoroute.c
index a2a64c6..ad02e82 100644
--- a/src/autoroute.c
+++ b/src/autoroute.c
@@ -5333,7 +5333,10 @@ donerouting:
     }
 #ifdef ROUTE_DEBUG
   if (ddraw != NULL)
-    ddraw->finish_debug_draw ();
+    {
+      ddraw->destroy_gc (ar_gc);
+      ddraw->finish_debug_draw ();
+    }
 #endif
 
   if (changed)
